src/ds/features/amplitude-factor-feature.h

```diff
@@ -0,0 +1,23 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <src/feature-interface.h>
+
+#include <string>
+
+
+namespace librealsense {
+
+class amplitude_factor_feature : public feature_interface
+{
+public:
+    static constexpr const char * ID = "Amplitude factor feature";
```

Comment:
If we keep these as `const char *`, then every feature lookup will involve a `strlen`, an allocation and copy to a `std::string`.